src/components/BlockchainList/ThreadPart.tsx

```diff
@@ -0,0 +1,85 @@
+import { classnames, display, flexDirection, gap } from 'classnames/tailwind'
+import CommentWithReplies from 'components/BlockchainList/CommentWithReplies'
+import Replies from 'components/BlockchainList/Replies'
+
+const wrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-y-3')
+)
+
+export default function ({ replyingTo }: { replyingTo: string }) {
+  const comments = [
```

Comment:
Just mock data, I hope it represents the real data, didn't find how it looks like in farcaster docs